--- b/aclocal.m4
+++ a/aclocal.m4
@@ -329,9 +329,7 @@
 AS_VAR_PUSHDEF([ac_enable],[enable_]m4_bpatsubst(m4_bpatsubst([$1],[.*/\([^/]*\)$],[\1]),[.*\.\(.*16\|vxd\)$],[win16]))dnl
 m4_append_uniq([_AC_USER_OPTS],ac_enable,[
 ])dnl
+m4_if(m4_bregexp([$1],[^tools]),[-1],[],[test "x$enable_tools" = xno || ])wine_fn_config_makefile [$1] ac_enable[]dnl
-m4_if(m4_bregexp([$1],[^tools]),[-1],[],
-      m4_if([$1],[tools/wine],[],[test "x$enable_tools" = xno || ]))dnl
-wine_fn_config_makefile [$1] ac_enable[]dnl
 AS_VAR_POPDEF([ac_enable])])
 
 dnl **** Append a file to the .gitignore list ****
--- b/configure
+++ a/configure
@@ -23948,7 +23948,7 @@
 test "x$enable_tools" = xno || wine_fn_config_makefile tools enable_tools
 test "x$enable_tools" = xno || wine_fn_config_makefile tools/sfnt2fon enable_sfnt2fon
 test "x$enable_tools" = xno || wine_fn_config_makefile tools/widl enable_widl
+test "x$enable_tools" = xno || wine_fn_config_makefile tools/wine enable_wine
-wine_fn_config_makefile tools/wine enable_wine
 test "x$enable_tools" = xno || wine_fn_config_makefile tools/winebuild enable_winebuild
 test "x$enable_tools" = xno || wine_fn_config_makefile tools/winedump enable_winedump
 test "x$enable_tools" = xno || wine_fn_config_makefile tools/winegcc enable_winegcc
